src/main/resources/static/codegen/built-api-spec.yml

```diff
@@ -0,0 +1,7411 @@
+          description: Only bookings for this service will be returned
+          schema:
+            $ref: '#/components/schemas/ServiceName'
+        - name: probationRegionId
+          in: query
+          required: false
+          description: If provided, only bookings for this region will be returned
+          schema:
+            type: string
+            format: uuid
+        - name: year
+          in: query
+          required: true
+          description: Only bookings for this year will be returned
+          schema:
+            type: integer
+        - name: month
+          in: query
+          required: true
+          description: Only bookings for this month will be returned - must be provided with year
+          schema:
+            type: integer
+      responses:
+        200:
+          description: Successfully retrieved the report
+          content:
+            'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet':
+              schema:
+                type: string
+                format: binary
+  /reports/bed-utilisation:
+    get:
+      tags:
+        - Reports
+      summary: Returns a spreadsheet showing how many days per month each bedspace spent in which state and the overall occupancy rate.
+      parameters:
+        - name: X-Service-Name
+          in: header
+          required: true
+          description: Only bookings for this service will be returned
+          schema:
+            $ref: '#/components/schemas/ServiceName'
+        - name: probationRegionId
+          in: query
+          required: false
+          description: If provided, only bookings for this region will be returned
+          schema:
+            type: string
+            format: uuid
+        - name: year
+          in: query
+          required: true
+          description: Only bookings for this year will be returned
+          schema:
+            type: integer
+        - name: month
+          in: query
+          required: true
+          description: Only bookings for this month will be returned - must be provided with year
+          schema:
+            type: integer
+      responses:
+        200:
+          description: Successfully retrieved the report
+          content:
+            'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet':
+              schema:
+                type: string
+                format: binary
+  /reports/lost-beds:
+    get:
+      tags:
+        - Reports
+      summary: Returns a spreadsheet listing all of the lost beds for the month
+      parameters:
+        - name: X-Service-Name
+          in: header
+          required: true
+          description: Only lost beds for this service will be returned
+          schema:
+            $ref: '#/components/schemas/ServiceName'
+        - name: probationRegionId
+          in: query
+          required: false
+          description: If provided, only lost beds for this region will be returned (CAS2)
```

Comment:
minor: It looks like CAS2 is mentioned in this description when it probably doesn't need to be?

Reply:
Well spotted! In this PR I've tried to limit changes to the structural to make the PR as clear as possible. i.e. this PR is "just" concerned with: 

- splitting the api.yml spec into two
- developing the Gradle tasks to generate Kotlin from 2 build artefacts

But yes, we certainly need to remove references like this to CAS2 in places where it no longer makes sense or is necessary. But I'd prefer to do that tidying in later PRs...